packages/calendar/src/Calendar.js

```diff
@@ -3,92 +3,136 @@ import Moment from 'moment';
 import DatePicker from './Datepicker/DatePicker';
 
 type
-	Props = {
-	/** The date format used to render the date. */
-	format?: string,
-	/** The selected or predefined date. */
-	activeDate?: string,
-	/** Every date in this prop will be selected and disabled */
-	selectedDates?: Array,
-	/** Every date less than this date will be disabled */
-	minDate: string,
-	/** Every date greater than this date will be disabled */
-	maxDate: string,
-	/** enable/disable the days during the weekend */
-	noWeekends?: boolean,
-	/** Event for when the date changes. */
-	onChange?: (e: object) => void,
-	/** Qa id */
-	qa?: string,
+Props = {
+    /** The date format used to render the date. */
+    format?: string,
+    /** The selected or predefined date. */
+    activeDate?: string,
+    /** Every date in this prop will be selected and disabled */
+    selectedDates?: Array,
+    /** Every date less than this date will be disabled */
+    minDate: string,
+    /** Every date greater than this date will be disabled */
+    maxDate: string,
+    /** enable/disable the days during the weekend */
+    noWeekends?: boolean,
+    /** Event for when the date changes. */
+    onChange?: (e: object) => void,
+    /** aria-label for previous, defaults to (viewType, value) => `Vorige ${viewType} (${value})` */
+    ariaLabelPrevious?: Function,
+    /** aria-label for next, defaults to (viewType, value) => `Volgende ${viewType} (${value})` */
+    ariaLabelNext?: Function,
+    /** aria-label for the datepicker, defaults to (viewType) => `Datumkiezer ${viewType}` */
+    ariaLabelDatePicker?: Function,
+    /** aria-label date format, defaults to 'dddd D MMMM YYYY' */
+    ariaLabelButtonDateFormat?: string,
+    /** Qa id */
+    qa?: string,
 };
 class Calendar extends Component<Prop> {
 
-	constructor(props) {
-		super(props);
-		const {activeDate, format} = this.props;
+    constructor(props) {
+        super(props);
+        const {activeDate, format} = this.props;
 
-		Moment.updateLocale('nl', {
-			months : ["januari", "februari", "maart", "april", "mei", "juni", "juli", "augustus", "september", "oktober", "november", "december"],
-			monthsShort: ["Jan", "Feb", "Mrt", "Apr", "Mei", "Jun", "Jul", "Aug", "Sep", "Okt", "Nov", "Dec"],
-			weekdaysShort: ["Ma", "Di", "Woe", "Do", "Vr", "Za", "Zo"]
-		});
+        Moment.updateLocale('nl', {
+                            months : [
+                                      "Januari",
```

Comment:
Maanden en dagen in het Nederlands zijn niet met een hoofdletter.

Reply:
Fixed.